tools/system_libs.py

```diff
@@ -74,7 +74,7 @@ def build_libc(lib_filename, files, lib_opts):
     # TODO: When updating musl the next time, feel free to recheck which of their warnings might have been fixed, and which ones of these could be cleaned up.
     c_opts = ['-Wno-return-type', '-Wno-parentheses', '-Wno-ignored-attributes', '-Wno-shift-count-overflow', '-Wno-shift-negative-value', '-Wno-dangling-else', '-Wno-unknown-pragmas', '-Wno-shift-op-parentheses', '-Wno-string-plus-int', '-Wno-logical-op-parentheses', '-Wno-bitwise-op-parentheses', '-Wno-visibility', '-Wno-pointer-sign']
     if shared.Settings.WASM_BACKEND:
-      c_opts.append('-Wno-error=absolute-value')
+      c_opts += ['-Wno-absolute-value', '-Wno-empty-body']
```

Comment:
looks good, but unless i'm missing something we should always append those two. the non-wasm backend case will need it too, eventually.

but ok to land this and decide that later if it's not obvious.

Reply:
Yes, you are right I think.  Not sure why the conditional is there at all TBH, but I didn't want to risk disturbing that build since its not broken right now.  (The only risk I guess would be that these arguments are not yet accepted by fastcomp clang?).   This seems like the safest change right now.

Reply:
Actually you are right .. I'll add these to both.